src/api/production-api-client.ts

```diff
@@ -164,3 +164,18 @@ export function createProductionApiClient(): ApiClient & LoggerApiClient {
 		searchIssues: productionApiClient.searchIssues.bind( productionApiClient ),
 	};
 }
+
+function safelySetLocalstorageCache( key: string, value: string ) {
+	try {
+		localStorage.setItem( key, value );
+	} catch ( err ) {
+		// Caching should never bubble up errors -- it's just a performance optimization!
+		// The most likely thing we'd hit when caching is a QuotaExceededError.
+		// So let's just leave a gentle console message about the user being able to add a perfomrance boost.
+		console.info(
+			`Unable to set cache value for ${ key }. ` +
+				`This likely means there is too much in localstorage for this domain. ` +
```

Comment:
**suggestion (non-blocking):** Replace 'domain' with 'website'. I know we love domains lately (lol), but I think using ´website´ here will be more user-friendly!

> This likely means there is too much in localstorage for this domain

Reply:
Bahaha, you're totally right! 😆 I'll get that updated.